woff1/tools/validator/validator.py

```diff
@@ -1792,7 +1792,7 @@ def calcPaddingLength(length):
     return 4 - (length % 4)
 
 def padData(data):
-    data += "\0" * calcPaddingLength(len(data))
+    data += "\0".encode() * calcPaddingLength(len(data))
```

Comment:
String data needs to be encoded in order to be able to perform binary data comparison in Python 3.